PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -135,7 +135,13 @@ open class PanModalPresentationController: UIPresentationController {
     private lazy var dragIndicatorView: UIView = {
         let view = UIView()
         view.backgroundColor = presentable?.dragIndicatorBackgroundColor
+        var alpha: CGFloat = 1.0
+        view.backgroundColor?.getRed(nil, green: nil, blue: nil, alpha: &alpha)
+        view.isOpaque = alpha == 1.0
         view.layer.cornerRadius = Constants.dragIndicatorSize.height / 2.0
+        if #available(iOS 13.0, *) {
+          view.layer.cornerCurve = .continuous
```

Comment:
Since this is a pill shape, I think we want the standard circular corners instead of continuous

Reply:
Does it make sense? Both are visually similar but the continuous one does the same as the current mask solution. Did you test my code?

Reply:
I think you're conflating switching to `cornerRadius` (which the `dragIndicatorView` already does) and specifying a `continuous` vs `circular` curve. A `continuous` curve is nice aesthetically when a corner is being rounded by itself. It creates a subtle "squircle" shape, rather than a simple circular radius that has a jarring transition to the flat part of the shape. This works great for something like the sheet view itself, so I support using `continuous` in that code.

However, in this drag view the two corners meet, and when you use the `continuous` style, this leads to point edges. The `circular` style has the desired effect, since it's two quarter circles that match perfectly. Here's a blown-up demonstration of the two effects when the radius is equal to the `height / 2`:

![Simulator Screen Shot - iPhone 12 Pro Max - 2021-01-29 at 12 12 18](https://user-images.githubusercontent.com/3526783/106323195-dc0eb800-622b-11eb-855e-61dbe89019c1.png)

I hope that make sense!

Reply:
I am pretty sure that the continous solution is still idea for this project.